src/pages/Product/Product.js

```diff
@@ -1,41 +1,61 @@
-import React from "react";
+import React, {useState, useEffect} from "react";
+import ColorButton from "../Product/components/ColorButton/ColorButton"
+import Color from "../Product/components/Color/Color"
+import Count from "../Product/components/Count/Count"
+import Review from "../Product/components/Review/Review"
 import "./Product.scss";
 
 const Product = () => {
-  const price = 300;
-  const totalPrice = price;
+  const [ colorChange, setColorChange ] = useState("white")
+  const [ count, setCount ] = useState(1)
+  const [ product, setProduct ] = useState([])
+  const price = 300
+  const totalPrice = price * count
 
+  useEffect(() => {
+    fetch('/data/product_info.json', {
+      method: 'GET'
+    }).then(res => res.json()).then(data => {
```

Comment:
```suggestion
    fetch('/data/product_info.json')
    .then(res => res.json())
    .then(data => {
```

GET 요청일 때, header나 body가 없다면 생략하셔도 좋습니다!